controllers/comments.controller.js

```diff
@@ -1,17 +1,48 @@
-const { fetchCommentsByReviewId, getAllReviewsHere } = require("../models/comments.model");
-
+const {
+  fetchCommentsByReviewId,
+  getAllReviewsHere,
+  addCommentByReviewId,
+  checkUserExists,
+} = require("../models/comments.model");
 
 exports.getCommentsByReviewId = (req, res, next) => {
   const { review_id } = req.params;
-     const promises = [getAllReviewsHere(review_id), fetchCommentsByReviewId(review_id)]
-    
-
+  const promises = [
+    getAllReviewsHere(review_id),
+    fetchCommentsByReviewId(review_id),
+  ];
 
   return Promise.all(promises)
-    .then(([irrelevant ,comments]) => {
+    .then(([irrelevant, comments]) => {
       res.status(200).send({ comments });
     })
     .catch((err) => {
       next(err);
     });
 };
+
+exports.postCommentByReviewId = (req, res, next) => {
+  const { review_id } = req.params;
+  const { username, body } = req.body;
+
+  if (
+    req.body.hasOwnProperty("username") === false ||
+    req.body.hasOwnProperty("body") === false
+  ) {
+    res.status(400).send({ msg: "Mandatory field missing" });
+  }
+
+  const promises = [
+    getAllReviewsHere(review_id),
+    checkUserExists(username),
```

Comment:
So this is a good idea. However we can avoid checking the username manually by just letting Postgres deal with it. What would happen if you removed this?